src/env-setter/env-setter.ts

```diff
@@ -0,0 +1,27 @@
+type validationFunctionType = (
+  localVariable: envType,
+  ...parameters: any[]
+) => boolean;
+type envType = string | undefined;
+
+export class EnvSetter {
+  static setPort(
+    localVariable: envType,
+    functionsToValidate: {
+      functionToValidate: validationFunctionType;
+      args?: any[];
+    }[]
+  ) {
+    let defaultPortValue;
+    functionsToValidate.some(({ functionToValidate, args }) => {
+      if (!functionToValidate(localVariable, args)) {
+        defaultPortValue = 3000;
+        console.log(
+          `${localVariable} doesn't meet required criteria for port\n setting default porty value: ${defaultPortValue}`
+        );
+        return true;
+      }
+    });
+    return defaultPortValue || Number(localVariable);
+  }
+}
```

Comment:
Pytanie: czy to jest setPort?
Jak na moje to jest ,,getPortValue" (nie ustawiasz tego nigdzie, tylko walidujesz)

Reply:
jednak się nie zgodzę, imho jest to ustawienie wartości